src/Compiler/TaskTableView.h

```diff
@@ -71,5 +71,7 @@ class TaskTableView : public QTableView {
  private:
   TaskManager *m_taskManager{nullptr};
   static constexpr uint TitleCol{1};
+  bool m_viewDisabled{
```

Comment:
Is it possible improve formatting? E.g.
`// comment here`
`bool m_viewDisabled{false};`